devtools/server/actors/replay/connection.js

```diff
@@ -944,9 +950,9 @@ function handleRecordingStarted(pmm) {
   // recording begins, so we lazily look it up the first time it is needed.
   let _browser = null;
   function getBrowser() {
-    for (let frameLoader of recordings.keys()) {
-      if (frameLoader.remoteTab && frameLoader.remoteTab.osPid === pmm.osPid) {
-        _browser = frameLoader.ownerElement;
+    for (const win of Services.wm.getEnumerator("navigator:browser")) {
```

Comment:
Why did this need to change? Having to iterate through all browsers seems really like something we should avoid.